### ISSUE TITLE
add Mastodon to Details page when editing a place

### ISSUE THREAD

**Issue by foss-** [organicmaps/organicmaps#5861]
**Is your feature request related to a problem? Please describe.**
Currently when editing a place the following social options show:

- Facebook
- Instagram
- X (Twitter)
- VK
- Line

**Describe the solution you would like**
Add Mastodon to that List which is supported as per https://wiki.openstreetmap.org/wiki/Mastodon

**Additional context**
Adding an open source and decentralized alternative to the current commercial and surveillance driven social networks would be highly appreciated.

One could debate whether the field should be named "Fediverse" and use logo https://commons.wikimedia.org/wiki/File:Fediverse_logo_proposal.svg or if it should indeed be "Mastodon". I'd assume if a shop uses another social network which is part of the fediverse, as a workaround, that info could also go into the mastodon field.

**Comment by RedAuburn** [comment]
Yeah, in theory i support this, but practically it would never be used. the average corner shop having a facebook account is rare, let alone a fedi account.

**Comment by foss-** [comment]
Keep in mind that e.g. in germany many municipal government bodies are switching into the Fediverse. And for those institutions it is much more common to have a Fediverse account than a Line account. Also I have not seen a single shop with Line but that may be very different in Japan.

**Comment by jpds** [comment]
https://www.swissinfo.ch/eng/business/swiss-authorities-to-join-social-media-alternative-mastodon/48808086 is also joining in.

A few users appear to have added a bunch of Mastodon accounts to OSM in the last month too.

**Comment by Coehill** [comment]
Any updates on this?

**Comment by VoxelPrismatic** [comment]
See #10142
